modules/installation-about-custom-gcp-vpc.adoc

```diff
@@ -38,7 +38,7 @@ If you destroy a cluster that uses an existing VPC, the VPC is not deleted.
 [id="installation-about-custom-gcp-permissions_{context}"]
 == Division of permissions
 
-Starting with {product-title} 4.3, you do not need all of the permissions that are required for an installation program-provisioned infrastructure cluster to deploy a cluster. This change mimics the division of permissions that you might have at your company: some individuals can create different resources in your clouds than others. For example, you might be able to create application-specific items, like instances, buckets, and load balancers, but not networking-related components such as VPCs, subnets, or ingress rules.
+Starting with {product-title} 4.3, you do not need all of the permissions that are required for an installation program-provisioned infrastructure cluster to deploy a cluster. This change mimics the division of permissions that you might have at your company: some individuals can create different resources in your clouds than others. For example, you might be able to create application-specific items, like instances, buckets, and load balancers, but not networking-related components such as VPCs, subnets, or Ingress rules.
```

Comment:
Are we treating _Ingress_ as capitalized?

Reply:
I pulled this from the [glossary](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#ingress), which seems to suggest keeping this uppercase.